code/modules/custom_ka/upgrade_chips.dm

```diff
@@ -2,8 +2,10 @@
 	name = "upgrade chip - damage increase"
 	desc = "Increases damage and recoil, reduces accuracy."
 	icon_state = "upgrade_chip"
-	damage_increase = 10
+	damage_increase = 15
 	recoil_increase = 3
+	firedelay_increase = 1.5 SECONDS
```

Comment:
Pretty substantial jump. Tested with multiple builds?

Reply:
As written above. The other point here is that it cannot be spammed against groups of carps, antags, the ISD or blobs, while still remaining useful. The gap between the different cores grows bigger with that, hopefully making people mix and match instead of going for the meta builds we have right now (if they want to use it for combat at all, the focus chip is a nice alternative, since it does not increase the firedelay ;) ).